src/app/technical-record/ministry-plates/ministry-plates.component.html

```diff
@@ -0,0 +1,26 @@
+<h2 class="govuk-heading-l">Select reason for sending ministry plate</h2>
+<form [formGroup]="platesForm">
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset">
+      <div class="govuk-radios">
+        <div *ngFor="let issue of issueValues; let i = index" class="govuk-radios__item">
+          <input
+            class="govuk-radios__input"
+            id="test-issue-{{ i }}"
+            type="radio"
+            formControlName="plateReasonForIssue"
+            value="{{ issue.includes('original') ? 'Original' : 'Replacement' }}"
+          />
```

Comment:
Let discuss this. but ideally you should covert your radio choices to options so you wont need to do this `value="{{ issue.includes('original') ? 'Original' : 'Replacement' }}"`